README.md

```diff
@@ -58,6 +58,32 @@ To build Mixxx, run
 There should now be a `mixxx` executable in the current directory that you can
 run. Alternatively, can generate a package using `cpack`.
 
+### Using Dev Container
+
+> [!NOTE]
+> Dev container has recently been introduced and it is likely incomplete! Currently, it's been tested with `devpod` on Zed and Codium exclusively
+
+We provide a DevContainer definition for Mixxx. The container base uses Ubuntu 24.04 to ensure a close similarity with the CI.
+Similar to Fedora Toolbox's, the default configuration provide a comprehensive definition of mounts which will allow use to use Mixxx seamlessly in the container. (UI, audio, ...)
+If you are not comfortable with this, you may want to remove some or all of the binding before starting the container.
+
+> [!TIP]
+> New to devcontainer? If you use VSCode, the IDE should offer you to use devcontainer when you open the Mixxx project. If you are not using VSCode,
+you may consider using [devpod](https://devpod.sh/docs/developing-in-workspaces/create-a-workspace#create-a-workspace)
+
+You can build and run Mixxx using the following command:
+
+```bash
+cd build
+cmake .. -DCMAKE_EXPORT_COMPILE_COMMANDS=ON # Needed for clangd
+cmake --build . -j $(nproc)
+./mixxx
+```
+
+#### Using your device in DevContainer
+
+Devices be supported by default, thanks to the provided binds. Note that if you are encountering permission issue, check that you are not using SELinux, as this is currently not well supported in DevContainer
```

Comment:
```suggestion
Devices be supported by default, thanks to the provided binds. Note that if you are encountering permission issue, check that you are not using SELinux, as this is currently not well supported in Dev Container
```

Reply:
Or remove the paragraph and move the note after "the container. (UI, audio, ...)" above.